EIPS/eip-5792.md

```diff
@@ -1,390 +1,269 @@
-that `eth_getTransactionReceipt` returns, excluding any fields that may differ across wallet implementations.
-
-#### `wallet_getBundleStatus` OpenRPC Specification
-
-```yaml
-- name: wallet_getBundleStatus
-  summary: Sends a bundle of function calls from the user wallet
-  params:
-    - name: Bundle identifier
-      required: true
-      schema:
-        type: string
-        title: Bundle identifier
-  result:
-    name: Call status
-    schema:
-      type: object
-      properties:
-        calls:
-          type: array
-          items:
-            title: call status
-            description: Status of the call at the given index
-            type: object
-            status:
-              title: The current status of the call
-              enum:
-                - CONFIRMED
-                - PENDING
-            receipt:
-              type: object
-              required:
-                - success
-                - blockHash
-                - blockNumber
-                - blockTimestamp
-                - gasUsed
-                - transactionHash
-              properties:
-                logs:
-                  type: array
-                  items:
-                    title: Log object
-                    type: object
-                    properties:
-                      address:
-                        $ref: '#/components/schemas/address'
-                      data:
-                        title: data
-                        $ref: '#/components/schemas/bytes'
-                      topics:
-                        title: topics
-                        type: array
-                        items:
-                          $ref: '#/components/schemas/bytes32'
-                success:
-                  type: boolean
-                  title: Whether the call succeeded
-                blockHash:
-                  title: The hash of the block in which the call was included
-                  $ref: '#/components/schemas/bytes32'
-                blockNumber:
-                  title: The number of the block in which the call was included
-                  $ref: '#/components/schemas/uint'
-                blockTimestamp:
-                  title: The timestamp of the block in which the call was included
-                  $ref: '#/components/schemas/uint'
-                gasUsed:
-                  title: How much gas the call actually used
-                  $ref: '#/components/schemas/uint'
-                transactionHash:
-                  title: The hash of the transaction in which the call was made
-                  $ref: '#/components/schemas/bytes32'
-```
+```typescript
+type GetCallsParams = string
 
-##### `wallet_getBundleStatus` Example Parameters
+type GetCallsResult = {
+  status: 'PENDING' | 'CONFIRMED'
+  receipts?: {
+    logs: {
+      address: `0x${string}`
+      data: `0x${string}`
+      topics: `0x${string}`[]
+    }[]
+    success: boolean
```

Comment:
Should we avoid flags like this? Should we align with the `status` in the receipt schema in the execution specs? https://github.com/ethereum/execution-apis/blob/main/src/schemas/receipt.yaml#L110-L113

Reply:
Aligning with status spec seems good. @lukasrosario?

Reply:
sgtm. will update here